agent_deploy/kubernetes/sysdig-agent-slim-daemonset-v1.yaml

```diff
@@ -0,0 +1,125 @@
+############################ DEPRECATION NOTICE ##########################
+# This daemonset yaml configures everything as an environment variable.
+# We recommend to use v2 daemonset which instead leverages Kubernetes
+# best practices like Secrets and ConfigMaps
+##########################################################################
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: sysdig-agent
+  labels:
+    app: sysdig-agent
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: sysdig-agent
+    spec:
+      volumes:
+      - name: dshm
+        emptyDir:
+          medium: Memory
+      - name: docker-sock
+        hostPath:
+          path: /var/run/docker.sock
+      - name: dev-vol
+        hostPath:
+          path: /dev
+      - name: proc-vol
+        hostPath:
+          path: /proc
+      - name: boot-vol
+        hostPath:
+          path: /boot
+      - name: modules-vol
+        hostPath:
+          path: /lib/modules
+      - name: usr-vol
+        hostPath:
+          path: /usr
+      hostNetwork: true
+      hostPID: true
+      tolerations:
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/master
+      ### OPTIONAL: If using OpenShift or Kubernetes RBAC you need to uncomment the following line
+      # serviceAccount: sysdig-agent
+      initContainers:
+      - name: sysdig-agent-kmodule
+        image: sysdig/agent-kmodule
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
+        resources:
+          # Resources needed are subjective on the actual workload
+          # please refer to Sysdig Support for more info about it
+          requests:
+            cpu: 100m
+            memory: 512Mi
+          limits:
+            memory: 1024Mi
+        volumeMounts:
+        - mountPath: /host/boot
+          name: boot-vol
+          readOnly: true
+        - mountPath: /host/lib/modules
+          name: modules-vol
+          readOnly: true
+        - mountPath: /host/usr
+          name: usr-vol
+          readOnly: true
+      containers:
+      - name: sysdig-agent
+        image: sysdig/agent-slim
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
+        resources:
+          # Resources needed are subjective on the actual workload
+          # please refer to Sysdig Support for more info about it
+          requests:
+            cpu: 100m
+            memory: 512Mi
+          limits:
```

Comment:
no limit on cpu?

Reply:
none of the other variants have a limit on cpu, as far as I know.